src/test-utils/dom/grid/index.ts

```diff
@@ -14,4 +15,17 @@ export default class GridWrapper extends ComponentWrapper<HTMLDivElement> {
   findColumn(columnIndex: number): ElementWrapper<HTMLDivElement> | null {
     return this.find(`.${styles['grid-column']}:nth-child(${columnIndex}) > div`);
   }
+
+  /**
+   * Returns the wrapper of the first column that matches the specified test ID.
+   * Looks for the `data-testid` attribute that is assigned via `gridDefinition` prop.
```

Comment:
Let's rather say that is looks for the testId property added to the gridDefinition.

Reply:
Like above comments, I don't think we should keep `data-testid` as an internal detail.